### ISSUE TITLE
Add Jellyfin Support

### ISSUE THREAD

**Issue by JasonHHouse** [JasonHHouse/gaps#142]
**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
Works like plex but for Jellyfin

**Describe alternatives you've considered**
None

**Additional context**
None

**Comment by elvis0288** [comment]
yes please, right now i use only jellyfin. i had to install plex just so i can see my gaps

**Comment by ankenyr** [comment]
Has there been any progress on this?
